app/controllers/api/functions.php

```diff
@@ -375,7 +375,7 @@
         }
 
         if ($deployment->isEmpty()) {
-            throw new Exception('Deployment not found', 404);
+            throw new Exception('Deployment not found', Exception::DEPLOYMENT_NOT_FOUND);
```

Comment:
```suggestion
            throw new Exception('Deployment not found', 404, Exception::DEPLOYMENT_NOT_FOUND);
```

At the moment, the `Exception` class still takes in the error code as the second parameter and the error type as the third parameter.  

In the next iteration, we'll derive the error codes from the error type as well. But for now, we still need 3 params